tutorials-and-examples/inference-servers/checkpoints/checkpoint_entrypoint.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
```

Comment:
perhaps name it `checkpoint_convertor.sh`?

Reply:
Done